plasmapy/simulation/particle_tracker.py

```diff
@@ -169,6 +170,89 @@ def total(self) -> float:
         return float(self.tracker.nparticles)
 
 
+class ParticlesPushedThroughGridsTerminationCondition(AbstractTerminationCondition):
+    """Termination condition corresponding to stopping the simulation when a provided
+    proportion of particles have entered and exited the grids.
+
+    Parameters
+    ----------
+    fraction_entered_threshold: float, optional
```

Comment:
So, the purpose of this line in the original tracker 

`if self.fract_entered <= 0.1`

Was just to make sure the tracker didn't terminate on the very first step (if no particle had yet traveled onto the grid enough to count as being on the grid). 

Maybe it's simpler to just remove this keyword and below make the condition 

```python
if self._particle_tracker.num_entered > 0 and self.fraction_exited_threshold < still_on:
    return True
```

So the termination condition basically turns on after the first particle gets to the grid? Then only one user-specified keyword "fraction_exited_threshold" should be enough.